fuzz/make_corpus.py

```diff
@@ -0,0 +1,23 @@
+import json
+import sys
+import os.path
+
+"""
+Creates a directory containing seed inputs from a json file.
+Usage:
+- pass the resulting corpus directory path as the first argument
+- pass the json file path to make the corpus from as the second argument
+"""
+corpus_dir = sys.argv[1]
+corpus_file_json = sys.argv[2]
```

Comment:
Also, all these things should go into functions or guarded by the typical `if __name__ == '__main__':`.